aodndata/srs/srs_oc_ljco.py

```diff
@@ -23,24 +23,39 @@ def dest_path(self, filepath):
                       netcdf_filename)
 
         if m is None:
-            raise ValueError("file name not matching regex to deduce dest_path")
+            m = re.search('^IMOS_SRS-OC-LJCO_(.*)_(.*)_LJCO_FV(.*)\.nc$', netcdf_filename)
```

Comment:
I think this whole regex matching/extracting bit can be cleaned up a lot. There are so many regex operations which look unnecessary, and I think the use of "named groups" would enhance readability a lot, and get rid of a lot of lines (remember, more lines = more bugs!).